src/components/screens/monitor/blocks/blocks.js

```diff
@@ -20,76 +25,102 @@ const columnClassNames = {
-                  <span className={columnClassNames.date}>
-                    <DateTimeFromTimestamp time={block.timestamp * 1000} token="BTC" />
-                  </span>
-                  <span className={columnClassNames.transactions}>
-                    {block.numberOfTransactions}
-                  </span>
-                  <span className={columnClassNames.generator}>
-                    <Link to={`${routes.accounts.pathPrefix}${routes.accounts.path}/${block.generatorAddress}`}>
-                      {block.generatorUsername}
-                    </Link>
-                  </span>
-                  <span className={columnClassNames.amount}>
-                    <LiskAmount val={block.totalAmount} />
-            &nbsp;
-                    {`${t('LSK')}`}
-                  </span>
-                  <span className={columnClassNames.forged}>
-                    <LiskAmount val={block.totalForged} />
-            &nbsp;
-                    {`${t('LSK')}`}
-                  </span>
+const Blocks = ({ t, blocks }) => {
+  const [filters, applyFilters, clearFilter, clearAllFilters] = useFilters(blocks, {
+    dateFrom: '',
+    dateTo: '',
+    height: '',
+    address: '',
+  });
+
+  const formatters = {
+    height: value => `${t('Height')}: ${value}`,
+    address: value => `${t('Generated by')}: ${value}`,
+  };
+
+  const handleLoadMore = () => {
+    blocks.loadData(Object.keys(filters).reduce((acc, key) => ({
+      ...acc,
+      ...(filters[key] && { [key]: filters[key] }),
+    }), {
+      offset: blocks.data.length,
+    }));
+  };
+
+  return (
+    <PageLayout>
+      <Box isLoading={blocks.isLoading} width="full" main>
+        <Box.Header>
+          <h1>{t('All blocks')}</h1>
+          <BlockFilterDropdown filters={filters} applyFilters={applyFilters} />
+        </Box.Header>
+        <FilterBar {...{
+          clearFilter, clearAllFilters, filters, formatters, t,
+        }}
+        />
+        { blocks.error
+          ? (
+            <Box.Content>
+              <Feedback status="error" show>{`${blocks.error}`}</Feedback>
+            </Box.Content>
+          )
+          : (
+            <React.Fragment>
+              <Box.Content className={styles.content}>
+                {!!blocks.data.length && (
+                <TableRow isHeader className={`${grid.row}`}>
+                  <div className={columnClassNames.id}>{t('ID')}</div>
+                  <div className={columnClassNames.height}>{t('Height')}</div>
+                  <div className={columnClassNames.date}>{t('Date')}</div>
+                  <div className={columnClassNames.transactions}>{t('Transactions')}</div>
+                  <div className={columnClassNames.generator}>{t('Generated by')}</div>
+                  <div className={columnClassNames.amount}>{t('Amount')}</div>
+                  <div className={columnClassNames.forged}>{t('Forged')}</div>
```

Comment:
Please extend the `columnClassNames` and add the label of the each column to that. Then you can use a loop to generate all of these divs.